integration-tests-support/test-support/src/main/java/org/apache/camel/quarkus/test/AvailablePortFinder.java

```diff
@@ -103,4 +124,19 @@ private static boolean isQuarkusReservedPort(int port) {
         }
         return false;
     }
+
+    private static String getCallerClassName() {
+        return StackWalker.getInstance(StackWalker.Option.RETAIN_CLASS_REFERENCE)
+                .walk(s -> s.map(StackWalker.StackFrame::getClassName)
+                        .filter(className -> !className.equals(AvailablePortFinder.class.getName()))
+                        .findFirst()
```

Comment:
So `findFirst()` returns the first class directly using AvailablePortFinder ? Would it work when we use AvailablePortFinder 2 times from the same class to reserve 2 distinct ports (if make sense) ?

Reply:
Ok, actually it's Map<int,String> so the value would appear 2 times in the map => OK